desk/lib/bolt.hoon

```diff
@@ -135,8 +133,8 @@
   ++  on-peek
     |=  =path
     ^-  (unit (unit cage))
-    ?+    -.path  (on-peek:ag path) 
-        %bolt  ~
+    ?+  path  (on-peek:ag path) 
+      [%x %bolt *]  ~
```

Comment:
Scries into Gall agents always start with the `care` used in the scry. By convention the care is always `%x`, though it could actually be whatever. You could change this to `[@ %bolt *]` if you wanted to future-proof things so that Bolt could accept e.g. `%y` scries in the future.